src/dosbox.cpp

```diff
@@ -812,10 +812,13 @@ void DOSBOX_Init() {
 	                  "           Use the filter of this Sound Blaster model.\n"
 	                  "  off:   Don't filter the output.");
 
-	Pstring = secprop->Add_string("cms_filter", when_idle, "on");
-	Pstring->Set_help("Filter for the Sound Blaster CMS output:\n"
+	const char *filter_on_or_none[] = {"on", "none", 0};
+
+	pstring = secprop->Add_string("cms_filter", when_idle, "on");
+	pstring->Set_help("Filter for the Sound Blaster CMS output:\n"
 	                  "  on:   Filter the output (default).\n"
 	                  "  off:  Don't filter the output.");
+	pstring->Set_values(filter_on_or_none);
```

Comment:
This should be rolled back because it breaks the filter code. For the filters `on` are `off` make more sense, it's like a toggle (I started with `none` myself, then changed it to `off`).

`none` makes sense for emulation modes, e.g. what you have for `pcspeaker` below (`discrete`, `impulse`, or `none`).